---
layout: page
title: software
description: Claudia Solis-Lemus' software
---

<div class="navbar">
    <div class="navbar-inner">
        <ul class="nav">
            <li><a href="https://github.com/crsl4">github-crsl4</a></li>
            <li><a href="https://github.com/solislemuslab">github-solislemuslab</a></li>
        </ul>
    </div>
</div>

---


<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://github.com/crsl4/PhyloNetworks.jl">PhyloNetworks</a><br/>
            A <a href="http://julialang.org/">Julia</a> package for statistical inference, data manipulation and visualization of phylogenetic networks<br/><br/>

            To receive announcements about updates to PhyloNetworks, join the<br/>
            <a href="https://groups.google.com/g/juliaphylo-users">JuliaPhylo google group</a><br/>

            Check out the tutorials: <br/>
            <a href="https://juliaphylo.github.io/PhyloUtilities/">PhyloUtilities website</a><br/> and the shorter and newer <br/>
            <a href="https://solislemuslab.github.io/snaq-tutorial/">SNaQ tutorial</a><br/>
        </div>

        <div class="span2">
        <a href="https://github.com/JuliaPhylo/PhyloNetworks.jl">
            <img src="../assets/pics/phylonetworks-logo.png"
                  title="PhyloNetworks" alt="PhyloNetworks"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://github.com/JuliaPhylo/SNaQ.jl">SNaQ</a><br/>
            Maximum pseudolikelihood estimation of species network. The procedure involves a numerical optimization of branch lengths and inheritance probabilities and a heuristic search in the space of phylogenetic networks.<br/><br/>

        </div>

        <div class="span2">
        <a href="https://github.com/JuliaPhylo/SNaQ.jl">
            <img src="../assets/pics/snaq.png"
                  title="SNaQ" alt="SNaQ"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://nathankolbow.github.io/InPhyNet.jl/stable/">InPhyNet</a><br/>
            A Julia package that merges several disjoint level-1 phylogenetic networks into one large network. This procedure significantly speeds up inference times for all sizes of networks without sacrificing accuracy.

            For an introduction to the software and a real-data example <a href="https://nathankolbow.github.io/InPhyNet.jl/stable/">see the documentation.</a><br/>
            <br/>Software developed by <a href="https://www.linkedin.com/in/nathankolbow/">Nathan Kolbow</a>.
        </div>

        <div class="span2">
            <a href="https://nathankolbow.github.io/InPhyNet.jl/stable/">
                <img src="../assets/pics/inphynet.png" title="InPhyNet-Logo" alt="InPhyNet-Logo"/>
            </a>
        </div>
    </div>
</div>

---

<a href="https://github.com/cecileane/iBPP/">iBPP</a><br/>
Bayesian species delimitation integrating genes and traits data.

---
<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://wi-fast-stats.wid.wisc.edu/">WI Fast Stats</a><br/>
            Integrated web page with access to all available web apps for data visualization of <a href="https://fastplants.org/">WI Fast Plants</a><br/><a href="https://youtu.be/6vX3mVXcK4s">YouTube short tutorial</a><br/><br/>
            Also please see the
            <a href="https://github.com/crsl4/fast-stats">github repo.</a><br/>
            <br/>Software developed by <a href="https://www.linkedin.com/in/yizhou-liu-539637199/">Yizhou Li.</a><br/>
        </div>

        <div class="span2">
        <a href="https://wi-fast-stats.wid.wisc.edu/">
            <img src="../assets/pics/fast-stats-logo-blue-1.png"
                  title="WI-Fast-Stats" alt="WI-Fast-Stats"/></a>
        </div>
    </div>
</div>

---


<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://github.com/YunyiShen/CAR-LASSO">CARlasso</a><br/>
            An R package for fitting Conditional Auto-Regressive LASSO, a class of models that infers (sparse) Gaussian chain graph with predictor and response nodes as well as its extensions<br/><br/>
            Check out the
            <a href="https://yunyishen.ml/CAR-LASSO/dev/">documentations and tutorials.</a><br/>
            <br/>Software developed by <a href="https://yunyishen.github.io/">Yunyi Shen.</a><br/>
        </div>

        <div class="span2">
        <a href="https://yunyishen.ml/CAR-LASSO/dev/">
            <img src="https://github.com/YunyiShen/CAR-LASSO/raw/dev/CARlasso.png"
                  title="CARlasso" alt="CARlasso"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://potato-seed-dashboard.wid.wisc.edu/">Potato Dashboard</a><br/>
            Proprietary data visualization and data analysis dashboard for the <a href="https://seedpotato.russell.wisc.edu/">WI Seed Potato Cert program</a><br/><br/>
            Check out the <a href="https://github.com/solislemuslab/potato-seed-cert">github repo.</a><br/>
            <br/>Software developed by <a href="https://github.com/hchen549">Haoming Chen</a>, <a href="https://www.linkedin.com/in/qiaowenwu/">Elaine Wu</a> and <a href="https://github.com/WXYS1209">Xiaoyang Wang.</a><br/>
        </div>

        <div class="span2">
        <a href="https://potato-seed-dashboard.wid.wisc.edu/">
            <img src="../assets/pics/potato.png"
                  title="WI-Seed-Potato" alt="WI-Seed-Potato"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://github.com/solislemuslab/BayesianNetworkRegression.jl">BayesianNetworkRegression.jl</a><br/>
            A Julia package for fitting Bayesian Network Regression, a model for inferring the relationships between microbiome networks and a dependent variable of interest like biological phenotypes.
            <br/>Check out the
            <a href="https://solislemuslab.github.io/BayesianNetworkRegression.jl/dev/">documentation and tutorials.</a><br/>
            <br/>Software developed by <a href="https://samozminkowski.com/">Sam Ozminkowski.</a><br/>
        </div>

        <div class="span2">
        <a href="https://solislemuslab.github.io/BayesianNetworkRegression.jl/dev/">
            <img src="../assets/pics/bayesiannetworkregressionjl.png"
                  title="BayesianNetworkRegression" alt="Bayesian network regression logo"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://bioklustering.wid.wisc.edu/">BioKlustering</a><br/>
            BioKlustering is a web app for learning and visualization of genomic data. You can choose from a variety of supervised, unsupervised and semi-supervised machine-learning methods to cluster genomic sequences.
            <br/>Check out the 
            <a href="https://github.com/solislemuslab/bioklustering">github repo.</a><br/>
            <br/>Software developed by <a href="https://www.linkedin.com/in/yuke-wu-8a38a8168/">Yuke Wu</a> and <a href="https://samozminkowski.com/">Sam Ozminkowski.</a><br/>
        </div>

        <div class="span2">
        <a href="https://bioklustering.wid.wisc.edu/">
            <img src="../assets/pics/bioklustering.png"
                  title="BioKlustering" alt="BioKlustering"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://github.com/solislemuslab/minaa">MiNAA</a><br/>
            MiNAA is a Microbiome Network Alignment Algorithm. While most existing alignment methods are tailored to protein or gene interaction networks, and often require particular biological data, MiNAA is suited to align microbiome interaction networks, and allows the user to include any biological data they see fit.<br/>
            <br/>Software developed by <a href="https://pages.cs.wisc.edu/~rnelson/">Reed Nelson.</a><br/>
        </div>

        <div class="span2">
        <a href="https://github.com/solislemuslab/minaa">
            <img src="../assets/pics/minaa.png"
                  title="MiNAA" alt="MiNAA"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://latinxinstat.wid.wisc.edu/">LatinxInStat Database</a><br/>
            The LatinxInStat database contains information of Latinx statisticians and data scientists. If you identify as Latinx, please add yourself to the database. If you are organizing a conference, please use this database to include a more diverse representation of speakers.<br/>
            <br/>Software developed by <a href="https://pages.cs.wisc.edu/~rnelson/">Reed Nelson.</a><br/>
        </div>

        <div class="span2">
        <a href="https://latinxinstat.wid.wisc.edu/">
            <img src="../assets/pics/latinx-stat.png"
                  title="LatinxInStat Satabase" alt="LatinxInStat Database"/></a>
        </div>
    </div>
</div>

<!-- UNCOMMENT WHEN FSD IS DEPLOYED -->
<!-- ---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://forensicstat.wid.wisc.edu/">Forensic-Statistic Database</a><br/>
            The Forensic-Statistic Database provides a list of statisticians and data scientists that are interested in aiding forensic scientists or attorneys in the understanding of statistical concepts in forensic science, or serving as experts in trials. If you are an expert willing to aid in such activities, please add yourself to the database.<br/>
            <br/>Software developed by <a href="https://pages.cs.wisc.edu/~rnelson/">Reed Nelson.</a><br/>
        </div>

        <div class="span2">
        <a href="https://forensicstat.wid.wisc.edu/">
            <img src="../assets/pics/forensic-stat.png"
                  title="Forensic-Satistic Database" alt="Forensic-Statistic Database"/></a>
        </div>
    </div>
</div> -->

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://solislemus-lab.shinyapps.io/sample-size-shinyapp/">Sample Size Calculation Shiny App</a><br/>
            This shiny app provides an open-source user friendly computation of sample size for different pairwise tests.<br/>
            <br/>Check out the 
            <a href="https://github.com/solislemuslab/sample-size-shinyapp">github repo.</a><br/>
            <br/>Software developed by <a href="https://github.com/nathanasiou">Nicole Athanasiou.</a><br/>
        </div>

        <div class="span2">
        <a href="https://solislemus-lab.shinyapps.io/sample-size-shinyapp/">
            <img src="../assets/pics/sample-size.png"
                  title="Sample Size Calculation Shiny App" alt="Sample Size Calculation Shiny App"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://github.com/solislemuslab/PhyloDiamond.jl/">PhyloDiamond.jl</a><br/>
            PhyloDiamond.jl is a Julia package for ultrafast learning of 4-node hybridization cycles in phylogenetic networks using algebraic invariants.<br/>
            <br/>Check out the 
            <a href="https://solislemuslab.github.io/PhyloDiamond.jl/dev/">documentation and tutorials.</a><br/>
            <br/>Software developed by <a href="https://github.com/zwu363">Zhaoxing Wu.</a><br/>
        </div>

        <div class="span2">
        <a href="https://github.com/solislemuslab/PhyloDiamond.jl">
            <img src="../assets/pics/phylodiamond_logo.png"
                  title="PhyloDiamond" alt="PhyloDiamond"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://github.com/solislemuslab/PhyloClustering.jl/">PhyloClustering.jl</a><br/>
            PhyloClustering.jl  is a Julia package to perform unsupervised learning on phylogenetic trees. 
            The algorithms currently included are K-means, Hierarchical Clustering, Gaussian Mixture Model, and DBSCAN.<br/>
            <br/>Check out the 
            <a href="https://solislemuslab.github.io/PhyloClustering.jl/dev/">documentation and tutorials.</a><br/>
            <br/>Software developed by <a href="https://github.com/YiboK">Yibo Kong.</a><br/>
        </div>

        <div class="span2">
        </div>
    </div>
</div>

---


<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://solislemus-lab.shinyapps.io/ShinyApp/">Virus Linker Shiny App</a><br/>
            The shiny app provides a platform for visualizing connections and interactions between proteins of viruses. <br/>
            <br/>Check out the 
            <a href="https://github.com/solislemuslab/virus-linker">github repo.</a><br/>
            <br/>Software developed by <a href="https://github.com/WXYS1209">Xiaoyang Wang.</a><br/>
        </div>

        <div class="span2">
        <a href="https://solislemus-lab.shinyapps.io/ShinyApp/">
            <img src="../assets/pics/virus_linker.png"
                  title="Virus Linker Shiny App" alt="Virus Linker Shiny App"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://github.com/solislemuslab/HighDimMixedModels.jl/">HighDimMixedModels.jl</a><br/>
            A Julia package for fitting regularized linear mixed-effects models on high-dimensional omics data <br/>
            <br/>Check out the 
            <a href="https://solislemuslab.github.io/HighDimMixedModels.jl/dev/">documentation and example usage.</a><br/>
            <br/>Software developed by <a href="https://sites.google.com/view/evangorstein">Evan Gorstein</a><br/>
        </div>

       <div class="span2">
        </div>
    </div>
</div>


---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://github.com/solislemuslab/CMiNet/">CMiNet</a><br/>
            CMiNet is an R package designed to generate consensus microbiome networks by integrating results from multiple network construction algorithms. This tool is specifically tailored for microbiome data, where capturing the intricate relationships between microbial taxa is essential to understanding complex biological systems.
            <br/><br/>
            The package employs a range of established algorithms, including:
            <ul>
                <li>Pearson and Spearman correlation</li>
                <li>Biweight midcorrelation</li>
                <li>Sparse Correlations for Compositional data (SparCC)</li>
                <li>Sparse InversE Covariance estimation for Ecological Association and Statistical Inference (SpiecEasi)</li>
                <li>Semi-Parametric Rank-based Correlation and Partial Correlation Estimation (SPRING)</li>
                <li>Generalized Co-Occurrence Differential Abundance analysis (gCoda)</li>
                <li>Correlation Inference for Compositional Data through Lasso (CCLasso)</li>
                <li>A novel algorithm based on conditional mutual information (CMIMN)</li>
            </ul>
            <br/>
            These algorithms construct individual microbial association networks, which CMiNet then combines into a single, weighted consensus network. By leveraging the strengths of each method, CMiNet provides a comprehensive and reliable representation of microbial interactions.
            <br/><br/>
            Software developed by <a href="https://www.linkedin.com/in/rosa-aghdam/">Rosa Aghdam</a><br/>
        </div>

        <div class="span2">
            <a href="https://github.com/solislemuslab/CMiNet/">
                <img src="../assets/pics/CMiNet.png"
                     title="CMiNet" alt="CMiNet R package"/></a>
        </div>
    </div>
</div>

---

<div class="container">
    <div class="row-fluid">
        <div class="span5">
            <a href="https://cminet.wid.wisc.edu" target="_blank">CMiNet</a><br/>
            <p>
                The CMiNet Shiny App is an interactive web-based tool that provides a user-friendly interface for generating consensus microbiome networks. 
                Built on the CMiNet R package, this app allows users to analyze microbiome data without requiring programming expertise, enabling intuitive visualization and exploration of microbial interactions.
            </p>
            <p>
                Software developed by <a href="https://www.linkedin.com/in/rosa-aghdam/" target="_blank">Rosa Aghdam</a>.
            </p>
        </div>
    </div>
</div>



